src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -68,11 +76,18 @@ class SQLPTRepresentationTest : public SQLPTRepresentation,
   static DBMS* dbms;
   static SQLPTRepresentation* reps;
   static const std::string kDatabaseName;
+  //Gtest can show message that this object doesn't destroyed
+  static utils::SharedPtr<policy_handler_test::MockPolicySettings> policy_settings_;
```

Comment:
@Kozoriz I don't like static SharedPtr. I don't think it's no other way. Try to change to something more simple.